app/controllers/v0/api_controller.rb

```diff
@@ -20,6 +20,7 @@ def resolve_version
 
       @version = {
         number: version.number,
+        id: version.id,
```

Comment:
Do we need this in response data?

Reply:
If not, [here's](https://github.com/department-of-veterans-affairs/gibct-data-service/pull/558) a refactor to separate a working `@version` object from the metadata.  With that you can just use `@version.id` wherever you need to, and `version` details in metadata stays the same.